tests/TestRvsdgs.cpp

```diff
@@ -331,8 +333,7 @@ Bits2PtrTest::SetupRvsdg()
   auto nf = graph->node_normal_form(typeid(jlm::rvsdg::operation));
   nf->set_mutable(false);
 
-  auto SetupBit2PtrFunction = [&]()
-  {
+  auto SetupBit2PtrFunction = [&]() {
```

Comment:
Looks like the curly brackets isn't following coding standard

Reply:
Overhauled the TestRvsdgs.hpp and TestRvsdgs.cpp